File: complex/A/graus.py
def converter(grau, referencia ):
	return (( grau * referencia ) / 360 )


def coordenar( horas, minutos ):	
	a = converter( horas, 12 )
	b = converter( minutos, 60 )
	za = str(a).zfill(2)
    zb = str(b).zfill(2)
    print "%02d" % a


while( True ):

    try:
        line = input()	
        mi = line.split('')
        horas = int(mi[0])
        minutos = int(mi[1])
        coordenar( horas, minutos );
    except EOFError:
        break		

File: complex/A/graus.java
package A;
import java.io.IOException;
import java.io.BufferedReader;
import java.io.FileReader;
import java.util.*;
 
public class Main {
    
    public static int converter(int grau, int referencia ){
	return (( grau * referencia ) / 360 );
    }

    public static void coordenar( int horas, int minutos ){
            int a, b;
            a = converter( horas, 12 );
            b = converter( minutos, 60 );
            System.out.printf("%02d:%02d\n", a, b);           
    }


    public static void main(String[] args) throws IOException {
        int h, m;
        Scanner input = new Scanner(System.in);
       try (FileReader reader = new FileReader("C:\\Users\\zsant\\Documents\\NetBeansProjects\\CMain\\src\\cmain\\graus.txt");
             BufferedReader br = new BufferedReader(reader)) {

            // read line by line
            String line;
            String[] parts;
            while ((line = br.readLine()) != null) {
                parts = line.split(" ");
                h = Integer.parseInt(parts[0]);
                m = Integer.parseInt(parts[1]);
                coordenar(h, m);                
            }

        } catch (IOException e) {
            System.err.format("IOException: %s%n", e);
        }        
    }
    
}
